src/back/controllers/api/index.js

```diff
@@ -3,4 +3,5 @@ const router = require('express').Router;
 
 module.exports = router()
     .use('/challenges', require('./routes/challenges'))
```

Comment:
i am a bit confusing with this **routes** folder controllers already means that there should be route handlers